base/applications/mspaint/imgarea.cpp

```diff
@@ -16,7 +16,7 @@
 
 /* FUNCTIONS ********************************************************/
 
-extern void
+extern BOOL
 zoomTo(int newZoom, int mouseX, int mouseY);
```

Comment:
If MSPaint has a common header, maybe it's time to move this there?

Reply:
Moved.